### PR TITLE
fix(mobile): Changes in the UI for the image editor pages

### PR DESCRIPTION
**Pull request by Yuvi-raj-P**
This commit brings many changes to the overall layout and user interface of the edit page and the crop page for the Immich mobile app. More Immich colors and better texts! Below are the changes that were made:

**NOTE: This commit required this commit to be merged first to work:**  https://github.com/immich-app/immich/pull/11741

| Before | After |
| ------------- | ------------- |
| <video src="https://github.com/user-attachments/assets/06b5ab0e-0842-4004-a2e1-4540ac702f26">  | <video src="https://github.com/user-attachments/assets/642e721f-f20f-4b6a-bcc9-7d63683f2e60">|

### CODE REVIEW

File: mobile/lib/pages/editing/crop.page.dart [hunk 1]
```diff
@@ -25,13 +25,14 @@ class CropImagePage extends HookWidget {
...
-        leading: CloseButton(color: Theme.of(context).iconTheme.color),
+        backgroundColor: Theme.of(context).scaffoldBackgroundColor,
+        title: const Text('Crop'),
+        leading: CloseButton(color: Theme.of(context).primaryColor),
         actions: [
           IconButton(
             icon: Icon(
               Icons.done_rounded,
-              color: Theme.of(context).iconTheme.color,
+              color: Theme.of(context).primaryColor,
```
Comment (alextran1502) [inline mobile/lib/pages/editing/crop.page.dart:35]: You can do `context.primaryColor` I believe

File: mobile/lib/pages/editing/crop.page.dart [hunk 2]
```diff
@@ -25,13 +25,14 @@ class CropImagePage extends HookWidget {
 
     return Scaffold(
       appBar: AppBar(
-        backgroundColor: Theme.of(context).bottomAppBarTheme.color,
-        leading: CloseButton(color: Theme.of(context).iconTheme.color),
+        backgroundColor: Theme.of(context).scaffoldBackgroundColor,
```
Comment (alextran1502) [inline mobile/lib/pages/editing/crop.page.dart:28]: Use `context.scaffoldBackgroundColor`

File: mobile/lib/pages/editing/crop.page.dart [hunk 3]
```diff
@@ -25,13 +25,14 @@ class CropImagePage extends HookWidget {
...
+        title: const Text('Crop'),
+        leading: CloseButton(color: Theme.of(context).primaryColor),
```
Comment (alextran1502) [inline mobile/lib/pages/editing/crop.page.dart:30]: `context.primaryColor`

File: mobile/lib/pages/editing/crop.page.dart (hunk 1 above)
Comment (Yuvi-raj-P) [reply mobile/lib/pages/editing/crop.page.dart:35]: Done!

File: mobile/lib/pages/editing/crop.page.dart (hunk 2 above)
Comment (Yuvi-raj-P) [reply mobile/lib/pages/editing/crop.page.dart:28]: Done!

**Comment by Yuvi-raj-P** [comment]
> Hello, please try to use `context.*` for theme related. Please also add localization to text string. For example `Text("localize_this".tr())`

Hi Alex! Yeah, I've never done localization before, so I don't really know about it on the Immich app. Sorry about that. It should be fixed now I guess! Please don't hesitate to let me know of any issues because it's my first time with the localization stuff.

**Comment by alextran1502** [comment]
@Yuvi-raj-P For localization, you only need to change/add the new key in the en-US.json file.

Thank you!

**Comment by Yuvi-raj-P** [comment]
😮‍💨 Glad that’s good! Will meet you with a new filters feature pr in 3 days Tanks!

**Comment by alextran1502** [comment]
@Yuvi-raj-P But you are modifying the `es-US.json` file that is for Spanish :P

**Review by alextran1502 — APPROVED** [review]

**Pull request merged**